modules/liveIntentAnalyticsAdapter.js

```diff
@@ -32,18 +32,18 @@ let liAnalytics = Object.assign(adapter({URL, ANALYTICS_TYPE}), {
   }
 });
 
-function handleAuctionInitEvent(auctionEndEvent) {
-  const liveIntentIdsPresent = checkLiveIntentIdsPresent(auctionEndEvent.bidderRequests)
+function handleAuctionInitEvent(auctionInitEvent) {
+  const liveIntentIdsPresent = checkLiveIntentIdsPresent(auctionInitEvent.bidderRequests)
 
   // This is for old integration that enable or disable the user id module
   // dependeing on the result of rolling the dice outside of Prebid.
-  const partnerIdFromAnalyticsLabels = auctionEndEvent.analyticsLabels?.partnerId;
+  const partnerIdFromAnalyticsLabels = auctionInitEvent.analyticsLabels?.partnerId;
 
   const data = {
     id: generateUUID(), // generated event id
-    aid: auctionEndEvent.auctionId, // auction id
+    aid: auctionInitEvent.auctionId, // auction id
     u: getRefererInfo().page, // page URL
-    ts: auctionEndEvent.timestamp, // timestamp of the auction
+    ts: auctionInitEvent.timestamp, // timestamp of the auction
```

Comment:
I changed the names of some fields in my PR. For auction-init I added a new field. I will help you to resolve the conflicts there.